core/domain/collection_services.py

```diff
@@ -949,23 +1184,39 @@ def get_next_page_of_all_non_private_commits(
 
 
 def _collection_rights_to_search_dict(rights):
-    # Allow searches like "is:featured".
+    """Allow searches like "is:featured".
```

Comment:
Hm, doesn't really say what this method does. Maybe:

Returns a search dict with information about the collection rights. This allows searches like "is:featured".